### PR TITLE
Added an "all games" admin page format, made some unit tests

### PR DESCRIPTION
**Pull request by kailbal**
Added a quick page for the "/games" route that shows all games. Used very similar code to what the "all users" view has, as these two items act very similarly to one another in what they want to accomplish.
Would probably be good for all admin.
Also has some minor changes from when I messed around with the "add games" features.
There are also some quick tests for the navigation menu.

### CODE REVIEW

**Comment by kailbal** [comment]
Have added new tests

File: frontend/src/test/navMenu.spec.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+// test the navigation menu component
+describe('NavMenu.vue', () => {
+  let wrapper;
+  beforeEach(() => {
+    wrapper = mount(NavMenu);
+  });
+  test('renders a navigation menu', () => {
+    //check if the navigation menu is rendered
+    expect(wrapper.find('v-list').exists()).toBe(true)
```
Comment (lynnbridge) [inline frontend/src/test/navMenu.spec.js:14]: I would actually check to see if the list item is rendered. I believe you can check for the title of the item.

File: frontend/package.json [hunk 2]
```diff
@@ -13,8 +16,15 @@
     "webfontloader": "^1.0.0"
```
Comment (lynnbridge) [inline frontend/package.json]: I believe you need to add asyncComputed to here unless it is included with vue3?

**Review by lynnbridge — COMMENTED** [review]
Please review my comments before I approve.

**Pull request merged**